### DISCUSSION TITLE
Greek Symbols for Seurat Cluster on Dimplot

### DISCUSSION THREAD

**Discussion by ShashTrip1** [satijalab/seurat#7140] [Q&A]
I am trying to label some γδ T cells and able to do so within the seurat object however when using DimPlot, it shows up as "   T Cells". 

Any suggestions?

**Comment by wudustan** [comment]
Seurat uses ggplot2 under the hood for plotting. So you will need to work with ggplot2 functionality to do this. There are [different methods](https://github.com/tidyverse/ggplot2/wiki/Plotmath) depending on which part of the plot you are working with. 

If it's just a one off label it might be worth exporting the PDF and editing it in Inksape or similar.

**Comment by wooyongc** [comment]
Hi, if anyone is looking for an answer, here's an easy solution.
Trick is to replace the greek letters you want with the unicode representation.

Hope it helped.

`seurat_obj <- RenameIdents(seurat_obj, 'gamma delta T cells' = "\u03B3\u03B4 T-cells")`

**Reply by wudustan** [reply]
Very elegant thanks for this!
